File: easy-solution/galaxy4276/70.java
class Solution {
  public int climbStairs(int n) {
    if (n == 1) return 1;
    if (n == 2) return 2;

    int mid, head, rear;
    mid = 1;
    head = 0;
    rear = 0;
    for (int i = 0; i < n; i++) {
      rear = mid + head;
      head = mid;
      mid = rear;
    }

    return rear;
  }
}

File: easy-solution/minesp3164/70.py
class Solution(object):
    def climbStairs(self, n):
        previewnum1 = 1
        previewnum2 = 2
        after =  0
        if n == 1:
            return 1
        if n == 2:
            return 2
        if n >= 3:
            for i in range(n - 2):
                after = previewnum1 + previewnum2
                previewnum1 = previewnum2
                previewnum2 = after
        return after

